articles/you-are-not-a-software-engineer.md

```diff
@@ -0,0 +1,58 @@
+---
+authorName: Stefan Djelekar
+authorGithubUsername: djelekar
+title: "You are not a Software Engineer" - What am I then?
+---
+# {{page.title}}
+
+The job of Software Engineer, or any technical role for that matter, is vast. You might spend more time than you would like on meetings, you might spend your afternoon sending emails to your stakeholders, you actually might only get to write code for a couple of hours a day. Depending on your personality type, this can make you thrilled or make you unhappy. I am writing this for the latter.
+
+This article is not about roles. Nor about daily duties. In this article, I offer a different view, in which our job is solving problems of business and making our customers happy, while coding is only a marginal tool in doing so. I maybe hit a nerve on this one, but it is even more important when working as a service provider. So hear me up.
+
+## "But I am a coder, I don't want to deal with any of this."
```

Comment:
Die Statements kommen etwas aprupt, ich hatte beim ersten Lesen Mühe, diese einzuordnen.
Zudem sehe ich im zweiten Abschnitt etwas wenig Inhalt. 

Wie wäre es, beim ersten Abschnitt folgendes anzuhängen und den zweiten Abschnitt ganz weg zu lassen?

I am writing this for the latter. In this article, I offer a view, in which our job is solving problems of business and making our customer happy, whily coding is only a marginal tool in doing so. 

oder gar

I am writing this for the latter, offering a different view, in which our job is solving problems of business and making our customer happy, whily coding is only a marginal tool in doing so.

Reply:
The statements below (title) come a bit apruptly. I had trouble putting them in order on the first read.
Additionally, I see a bit little content in the second paragraph.

What about adding the following to the first paragraph and leavign the second one out?
I am writing this for the latter. In this article, I offer a view, in which our job is solving problems of business and making our customer happy, whily coding is only a marginal tool in doing so.
or even
I am writing this for the latter, offering a different view, in which our job is solving problems of business and making our customer happy, whily coding is only a marginal tool in doing so.

Reply:
Agree. I will go for the first approach, as the sentence might be to big in the second proposal. Also I would like to emphasize - "I'm writing this for the latter."